### DISCUSSION TITLE
Something went wrong. Please try again a bit later.

### DISCUSSION THREAD

**Discussion by DariusArya** [umputun/remark42#1141] [Q&A]
I set it up and comments with editor window appeared. 
It runs on docker, on Ubuntu linux
log in works with google for example. 
but when I write a test comment and submit 
I get this 
  Something went wrong. Please try again a bit later.
  
  But my question is this: 

  Is there an error log somewhere that I can consult to find out what's happening

**Comment by umputun** [comment]
pls use [discussions](https://github.com/umputun/remark42/discussions) for this sort of Q&A and other things that are not the actual issues. I'll move it for you

**Comment by umputun** [comment]
> Is there an error log somewhere that I can consult to find out what's happening

sure. you have the standard docker facility to access logs, i.e. docker log or docker-compose logs

**Reply by DariusArya** [reply]
thank you,  I am new to docker, thank you for your help.
